src/controllers/itemDetails/index.js

```diff
@@ -167,7 +167,30 @@ function renderTrackSelections(page, instance, item, forceReload) {
         return;
     }
 
-    const mediaSources = item.MediaSources;
+    var mediaSources = item.MediaSources;
+
+    var resolutionNames = [];
+    var sourceNames = [];
+    mediaSources.forEach(function (v) {
+        (v.Name.endsWith('p') || v.Name.endsWith('i')) ? resolutionNames.push(v) : sourceNames.push(v);
```

Comment:
There should probably be a check here to make sure that `parseInt(v.Name, 10)` returns a number and if not also push that to `sourceNames`. Something like:
```js
(v.Name.endsWith('p') || v.Name.endsWith('i')) && !isNaN(parseInt(v.Name, 10))
```